project/AppDependencies.scala

```diff
@@ -26,8 +26,7 @@ object AppDependencies {
     "com.vladsch.flexmark"     %  "flexmark-all"            % "0.35.10"  % "test, it",
     "org.scalatestplus.play"   %% "scalatestplus-play"      % "3.1.2"    % "test, it",
     "org.scalatestplus"        %% "scalacheck-1-14"         % "3.2.2.0"  % Test,
-    "org.scalatestplus"        %% "selenium-3-141"          % "3.2.2.0"  % Test,
-    "com.softwaremill.macwire" %% "macros"                  % "2.3.7"    % Test
+    "org.scalatestplus"        %% "selenium-3-141"          % "3.2.2.0"  % Test
```

Comment:
the test jar `bootstrap-test-play-27` still out of date ? 
remove `jsoup` if its not needed anymore, also the `flexmark`

Reply:
bootstrap has been updated to 28, jsoup removed.  
I'm unable to find out what part of the build is using flexmark. It is being used by scalatest to create HTML reports but I'm so far unable to determine why.